bundle-generator/runtime/src/main/java/io/quarkiverse/operatorsdk/bundle/runtime/CSVMetadataHolder.java

```diff
@@ -6,17 +6,50 @@ public class CSVMetadataHolder {
     public final String name;
     public final String description;
     public final String displayName;
+    public final Annotations annotations;
     public final String[] keywords;
     public final String providerName;
     public final String providerURL;
     public final String replaces;
     public final String version;
     public final String maturity;
+    public final String minKubeVersion;
     public final Maintainer[] maintainers;
+    public final Link[] links;
+    public final Icon[] icon;
```

Comment:
Seems like this fields would be better as a plural as well…

Reply:
I intentionally used singular for `icon` (despite being an array) to reflect ClusterServiceVersion tag `icon` that is singular as well
<img width="157" alt="Screen Shot 2022-10-21 at 9 11 49 AM" src="https://user-images.githubusercontent.com/1379213/197203916-b3f36004-6e2e-4ed6-96ea-3ed0a3c267a0.png">

Reply:
Does 👍 mean you are agree and I could close it as resolved?

Reply:
Yes.